### ISSUE TITLE
Patch comments

### ISSUE THREAD

**Issue by brunomiranda-hotmart** [suzuki-shunsuke/tfcmt#199]
Instead of using github-comment to hide comments, would be nice to update comments.
This makes the PR time less verbose and only reflects what is currently being applied.

![image](https://user-images.githubusercontent.com/84582602/152033126-c35c8163-7c43-4edf-83ef-6632d33abda9.png)

(Might be an issue with https://github.com/suzuki-shunsuke/github-comment/ as well)

**Comment by guidodobboletta** [comment]
This would be a very nice to have. Tools like https://www.infracost.io/ have that feature in which they update the comment in place instead of hiding it.

**Comment by anthr76** [comment]
This functionality is available with `peter-evans/find-comment@v2` for some inspiration

**Comment by suzuki-shunsuke** [comment]
Thank you for your proposal.

**Comment by suzuki-shunsuke** [comment]
# Proposal: Add `-patch` option to `tfcmt plan` command

```console
$ tfcmt plan -patch [-var "target:<target>"] -- terraform plan
```

* List pull request comments
* Get the patched pull request comment with the comment metadata `target`
  * If multiple comments are matched, the first comment is patched
* Patch the comment. If no comment is matched, Create a comment

**Comment by suzuki-shunsuke** [comment]
#245

**Comment by akhan4u** [comment]
This looks shiny ✨ Looking forward to this option.

**Comment by suzuki-shunsuke** [comment]
I've published the prerelease version.

* https://github.com/suzuki-shunsuke/tfcmt/releases/tag/v3.2.0-0
* https://github.com/suzuki-shunsuke/tfaction/releases/tag/v0.5.10-0

It seems to work well, but I'll verify it for a while.

**Comment by suzuki-shunsuke** [comment]
https://github.com/suzuki-shunsuke/tfcmt/pull/249

**Comment by suzuki-shunsuke** [comment]
Released [v3.2.0](https://github.com/suzuki-shunsuke/tfcmt/releases/tag/v3.2.0) 🎉 
https://suzuki-shunsuke.github.io/tfcmt/plan-patch

**Comment by suzuki-shunsuke** [comment]
Bug Fix https://github.com/suzuki-shunsuke/tfcmt/pull/253
